core/sr-primitives/src/generic/digest.rs

```diff
@@ -15,6 +15,59 @@
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
 //! Generic implementation of a digest.
+//!
+//! There are three k
+//! _Inherent digests_ are digests that are provided to the runtime by the
+//! consensus algorithm. They are inherent because the runtime trusts them
+//! implicitly, although the consensus algorithm does verify them.
+//!
+//! This PR passes the digests in the header to the runtime via
+//! `initialize_block`. It does _not_, however, provide any means for runtimes
+//! to actually use the information. This will be provided later.
```

Comment:
Why is here some sort of changelog? This is code documentation and not a place for a changelog.